MantidQt/CustomInterfaces/src/Indirect/Elwin.cpp

```diff
@@ -217,22 +218,14 @@ void Elwin::run() {
 
   m_batchAlgoRunner->addAlgorithm(elwinMultAlg, elwinInputProps);
 
-  // Configure Save algorithms
-  if (m_uiForm.ckSave->isChecked()) {
-    addSaveAlgorithm(qWorkspace);
-    addSaveAlgorithm(qSquaredWorkspace);
-    addSaveAlgorithm(elfWorkspace);
-
-    if (m_blnManager->value(m_properties["Normalise"]))
-      addSaveAlgorithm(eltWorkspace);
-  }
-
   connect(m_batchAlgoRunner, SIGNAL(batchComplete(bool)), this,
           SLOT(unGroupInput(bool)));
   m_batchAlgoRunner->executeBatchAsync();
 
   // Set the result workspace for Python script export
   m_pythonExportWsName = qSquaredWorkspace;
+
+  m_elwinAlg = elwinMultAlg;
```

Comment:
Fine, but we could have achieved the same functionality just by renaming `elwinMultAlg` to `m_elwinAlg`, then there's no need to define an additional variable to store the algorithm.